contrib/babelfishpg_tsql/src/hooks.c

```diff
@@ -5864,3 +5870,44 @@ is_bbf_db_ddladmin_operation(Oid namespaceId)
 
 	return false;
 }
+
+/*
+ * remove_db_name_in_schema - remove the db name and underscore at the beginning
+ * 	of the given string. It is used to unmap schema name in error messages.
+ *
+ * 	@param schema_name - char *
+ * 	@return - unmapped schema name char *
+ */
+static const char *
+remove_db_name_in_schema(const char *object_name)
+{
+	const char	*cur_db_name;
+	char		**splited_object_name;
+	char		*schema_name = NULL;
+	char		*mutable_name;
+    size_t 		db_name_len;
+    size_t 		prefix_len;
+	size_t 		schema_name_len;
+
+	mutable_name = strdup(object_name);
+	splited_object_name = split_object_name(mutable_name);
+	for (int i = 0; i < 4; i++)
+	{
+		if (strcmp(splited_object_name[i], "")) {
```

Comment:
I think our code checker will complain about `strcmp` usage. I would suggest either `strncmp` or in this case, maybe you can just use a `strlen` call.